gst-plugin-pravega/src/pravegasink/imp.rs

```diff
@@ -281,6 +375,31 @@ impl ObjectImpl for PravegaSink {
                 None,
                 glib::ParamFlags::WRITABLE,
             ),
+            glib::ParamSpec::new_string(
+                PROPERTY_NAME_RETENTION_TYPE,
+                "Retention type",
+                "The retention policy for video stream. Can be days, size, daysAndBytes or none.",
+                None,
+                glib::ParamFlags::WRITABLE,
+            ),
+            glib::ParamSpec::new_double(
+                PROPERTY_NAME_RETENTION_DAYS,
+                "Retention days",
+                "The number of days that the video stream will be retained.",
+                std::f64::NEG_INFINITY,
```

Comment:
Why NEG_INFINITY for the min? I'm confused why we would accept a negative value here.

Reply:
fixed